multigeiger/loraWan.cpp

```diff
@@ -96,59 +117,54 @@ const lmic_pinmap lmic_pins = {
     .dio = {LORA_IRQ, LORA_IO1, LORA_IO2 },
 };
 
-static volatile transmitionStatus_t txStatus;
+static volatile transmissionStatus_t txStatus;
 static uint8_t * __rxPort;
 static uint8_t * __rxBuffer;
 static uint8_t * __rxSz;
 
+void event_log(const char *event){
+  log(DEBUG, "%s: %s", os_getTime(), event);
```

Comment:
os_getTime doesn't return a char*, it returns ostime_t. so it  should be  
log(DEBUG, "%d: %s", os_getTime(), event);
Question: why do you read and show the time here?

Reply:
the code already was like that, I only changed the direct call to Serial.print* to log(...).

Reply:
fixing the format...

Reply:
fixed